frontend/src/pages/home/Home.tsx

```diff
@@ -43,7 +44,8 @@ const Home: React.FC = () => {
       ) : (
         <>
           <ProjectsSection />
-          <ModelCatalogSection />
+          {/* TODO: Temporarily disabled model catalog section  */}
+          {/* <ModelCatalogSection /> */}
```

Comment:
Let's link to this issue in this comment: https://issues.redhat.com/browse/RHOAIENG-34405